src/translator/service.h

```diff
@@ -84,7 +84,7 @@ class Service {
   explicit Service(const std::string &config,
                    AlignedMemory modelMemory = AlignedMemory(),
                    AlignedMemory shortlistMemory = AlignedMemory())
-      : Service(parseOptions(config), std::move(modelMemory),
+      : Service(parseOptions(config, /*validate=*/false), std::move(modelMemory),
```

Comment:
The comment here suggests the default is false when it's true.

Reply:
@XapaJIaMnu Do you want me to remove `/*validate=*/` from here?